src/client/utils/validations.js

```diff
@@ -267,10 +267,10 @@ function isValidFinancialDisclosure(data) {
   return validateIfDirty(data.understandsFinancialDisclosure, isNotBlank);
 }
 
-function isValidIncome(income) {
-  return isValidField(isValidMonetaryValue, income.grossIncome) &&
-      isValidField(isValidMonetaryValue, income.netIncome) &&
-      isValidField(isValidMonetaryValue, income.otherIncome);
+function isValidRequiredIncome(income) {
+  return isValidRequiredField(isValidMonetaryValue, income.grossIncome) &&
```

Comment:
Hm... I don't think so, since it's a higher-level check, and we already call it [here](https://github.com/department-of-veterans-affairs/healthcare-application/blob/8d49d9b4bdd97a8963208314147ce63febe325bb/src/client/utils/validations.js#L347)